src/assessment.js

```diff
@@ -49,5 +55,5 @@ var q2 = [/* your answer here */];
 // executing it.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  deathStar.fire.bind(alderaan);
```

Comment:
So close! If you use `bind` you need to invoke it in order for the newly-modified function to run.
e.g.
`deathStar.fire.bind(alderaan)()`